articles/rich-text-inputs.mdx

```diff
@@ -118,44 +174,20 @@ You'll likely need to convert the Markdown value into HTML to use it on your sit
 
```

Comment:
Fixup for the content that was already there: we should avoid using "Other SSGs" since it doesn't make sense when read on the page. Maybe just dropping the `Other `